### ISSUE TITLE
Browserify root import (preventing relative import)

### ISSUE THREAD

**Issue by Allov** [benmosher/eslint-plugin-import#517]
Maybe I'm blind, but I can't seem to find a way to setup Browserify's paths to the resolver. Here's my eslintrc file:

**.eslintrc**
```
{
    "extends": [
        "eslint:recommended",
        "plugin:import/errors",
        "plugin:import/warnings"
    ],
    "env": {
        "browser": true,
        "es6": true
    },
    "parser": "babel-eslint",
    "globals": {
        "Phaser": true
    },
    "rules": {
        "strict": 0,
        "indent": [2, 4],
        "max-len": [2, 200, 2,
            {
                "ignoreUrls": true,
                "ignoreComments": true
            }],
        "quotes": [2, "single", "avoid-escape"],
        "no-shadow": 1
    }
}
```

I'm using gulp to initialize Browserify this way:

**gulpfile.js**
```javascript
    return browserify({
            paths: [path.join(__dirname, 'src')],
            entries: ENTRY_FILE,
            debug: true,
            transform: [
                [
                    babelify, {
                        presets: ["es2015"]
                    }
                ]
            ]
        })
```

But, whenever I import something, eslint reports that it can't find the file (there are no issue when executing the code):

**Folder structure:**
```
src
   |- objects
      |- ColoredText.js
   |- states
      |- GameState.js
   |- index.js
.eslintrc
gulpfile.js
```

**import code in GameState.js:**
```javascript
import ColoredText from 'objects/ColoredText';
```

**package.json for reference**
```
  "devDependencies": {
    "babel-eslint": "^6.1.2",
    "babel-preset-es2015": "^6.13.2",
    "babelify": "^7.2.0",
    "browser-sync": "^2.14.0",
    "browserify": "^13.1.0",
    "del": "^2.2.2",
    "eslint": "^3.3.1",
    "eslint-config-import": "^0.13.0",
    "eslint-plugin-import": "^1.14.0",
    "exorcist": "^0.4.0",
    "gulp": "^3.9.1",
    "gulp-buffer": "0.0.2",
    "gulp-if": "^2.0.0",
    "gulp-uglify": "^1.5.3",
    "gulp-util": "^3.0.7",
    "vinyl-source-stream": "^1.1.0",
    "yargs": "^4.6.0"
  },
  "dependencies": {
    "phaser": "^2.6.1"
  }
```

**Comment by benmosher** [comment]
You might need: 

```js
"settings" : {
  "import/resolver": {
    "node": { "paths": [path.join(__dirname, 'src')] }
  }
}
```
in your `.eslintrc.js`. (if it's json, just `"paths": ["./src"]` should work, as long as you always run ESLint from the same folder)

**Comment by benmosher** [comment]
(side note: `eslint-config-import` is totally obviated by v1, you can remove it from your devDeps)

**Comment by Allov** [comment]
yay! thanks, it's working!
